frontend/src/components/GoogleLogin.vue

```diff
@@ -6,7 +6,7 @@
       href="#"
       @click="signOut"
     >SIGN OUT</a>
-    <div id="signin2" />
+    <div v-else id="signin2" />
```

Comment:
You dont need this in the new and improved NavbarComponent B)
Also the stuff above that one is not needed in the new and improved NavbarComponent B)
```suggestion
    <div id="signin2" />
```

Reply:
okay nice